tests/models/gemma2/test_modeling_gemma2.py

```diff
@@ -209,7 +211,7 @@ def test_eager_attention_loaded_by_default(self):
 
 
 @slow
-@require_torch_gpu
+# @require_torch_gpu
```

Comment:
to revert!

Reply:
Aha, forget to change this back. Good catch!